libs/errors/website/src/lib/apollo-error-alert.tsx

```diff
@@ -0,0 +1,55 @@
+import {ApolloError} from '@apollo/client/errors'
+import {ComponentProps, PropsWithChildren, useMemo} from 'react'
+import {ErrorCode} from 'errors'
+import {Alert as MuiAlert, Alert} from '@mui/material'
+import {Link} from '@wepublish/ui'
```

Comment:
cannot import Link via useWebsiteBuilder() function, because this would end up in circular dependencies. Now this link is reloading the whole page. Any idea how to avoid that @Itrulia ?

Reply:
By importing useWebsiteBuilder from @wepublish/website/builder then it works